content/docs/dev/concepts/explore-tikv-features/backup-restore.md

```diff
@@ -10,12 +10,156 @@ menu:
 
 ## RawKV Backup & Restore
 
-*TODO*
+**[TiKV Backup & Restore (TiKV-BR)](https://github.com/tikv/migration/tree/main/br)** is a command-line tool for distributed backup and restoration of the TiKV cluster data.
 
-## Instruction Manual
+## Architecture
 
-*TODO*
+{{< figure
+    src="/img/docs/tikv-br.png"
+    caption="TiKV-BR architecture"
+    number="1" >}}
 
-## Usage Demonstration
 
-*TODO*
+### Recommended Deployment Configuration
+- In production environments, deploy `TiKV-BR` on a node with at least 8 cores CPU and 16 GB memory. Select an appropriate OS version by following [Linux OS version requirements](https://docs.pingcap.com/tidb/dev/hardware-and-software-requirements#linux-os-version-requirements).
+
+- Save backup data to Amazon S3 or one mounted network disk on all `TiKV-BR` and `TiKV` nodes.
+
+- Allocate sufficient resources for backup and restoration.
+
+- TiKV-BR only support raw data backup/restoration in TiKV cluster with version >= `5.0.0`.
+
+TiKV-BR, TiKV nodes, and the backup storage system should provide network bandwidth that is greater than the backup speed. If the target cluster is particularly large, the threshold of backup and restoration speed is limited by the bandwidth of the backup network.  
```

Comment:
```suggestion
TiKV-BR, TiKV nodes, and the backup storage system should provide network bandwidth that is greater than the backup speed. If the target cluster is particularly large, the backup and restoration speed highly depends on the bandwidth of storage system bandwidth.  
```

Reply:
Addressed and apply some suggestions from google docs, PTAL, thanks.